pkg/controller/master/storagenetwork/storage_network.go

```diff
@@ -596,11 +596,9 @@ func (h *Handler) checkPodStatusAndStart() error {
 		return err
 	}
 
-	if err := h.checkVMImportControllerStatusAndStart(); err != nil {
-		return err
-	}
+	err := h.checkVMImportControllerStatusAndStart()
 
-	return nil
+	return err
```

Comment:
prefer to `return h.checkVMImportControllerStatusAndStart()`

Reply:
I will consider to keep the original format (before codefactor commit)
but I will change it.